spec/models/article_spec.rb

```diff
@@ -8,14 +8,19 @@
     it { is_expected.to have_db_column(:theme).of_type(:string) }
     it { is_expected.to have_db_column(:backyard).of_type(:boolean) }
     it { is_expected.to have_db_column(:premium).of_type(:boolean) }
-    it { is_expected.to have_db_column(:published).of_type(:boolean) }
+
```

Comment:
We cant test if it has `(:status).of_type(:integer)` ?

Reply:
we can, but on line 16 we have `is_expected.to define_enum_for(:status).with_values({ archived: 1, draft: 5, published: 10 }) }` isn't that double assertion?